core/controllers/editor_test.py

```diff
@@ -697,6 +702,73 @@ def test_deletion_rights_for_published_exploration(self):
         self.assertEqual(response.status_int, 200)
         self.logout()
 
+    def test_logging_info_after_deletion(self):
+        """Tests correctness of logged statements while deleting exploration"""
+        logging_info = []
+
+        def add_logging_info(msg, *_):
```

Comment:
Yes.
logging.info can take two arguments. So, when it is called from other part of codebase like 'all_pending: clear %s' in below case or in 'Attempting to delete documents from index %s, ids: %s' % (index.name, ', '.join(doc_ids)), it is passed two arguments.
That is why we need to have second arg to handle these.